hack/update-codegen-dockerized.sh

```diff
@@ -41,12 +41,14 @@ $GOPATH/bin/client-gen \
 # Generate listers with K8s codegen tools.
 $GOPATH/bin/lister-gen \
   --input-dirs "${ANTREA_PKG}/pkg/apis/security/v1alpha1,${ANTREA_PKG}/pkg/apis/core/v1alpha1" \
```

Comment:
Do you mean we can merge all input-dirs into one line? If we may add new listers/informers in future, we can keep this format because too long line is not readable.

Reply:
that is how it is done currently.. but i don't have a strong preference to continue doing that.. so you could ignore